Fix landmark-unique error due to in-page TOC (#1607)
pydata/pydata-sphinx-theme: docs/examples/in-page-toc.rst

:html_theme.sidebar_secondary.remove: true

Test of in-page TOC with no right sidebar
=========================================

This page tests that the local contents directive looks okay.

We **do not recommend** using this directive on pages that use this theme
because PyData Theme provides an in-page table of contents in the right sidebar
by default.

> If you do choose to use an inline, in-page table of contents, we recommend that
> you turn off the right sidebar (directions below).

Reviewer: ```suggestion
If you do choose to use an inline, in-page table of contents, we recommend that
you turn off the right sidebar as follows.
```

.. contents:: Page contents
    :local:

> Local contents directive
> ------------------------
> 
> What is a local contents directive?
> 
> .. code-block:: rst
> 
>     .. contents:: Page contents
>         :local:
> 
> You add this directive near the top of your ``.rst`` page to insert a table of
> contents for that page, which is what this page does.
> 

Reviewer: ```suggestion
Add a local table of contents (in-page)
---------------------------------------

Add the local table of contents directive near the top of your ``.rst`` page: 

.. code-block:: rst

    .. contents:: Page contents
        :local:

This directive will generate a table of contents for the section where this was added, as shown on this page.

```

The previous wording was a bit confusing

> Turn off right sidebar for a single page
> ----------------------------------------

Reviewer: ```suggestion
Turn off the right sidebar for a single page
--------------------------------------------
```
